ts/utils/hooks/bottomSheet.tsx

```diff
@@ -168,3 +175,62 @@ export const useIOBottomSheetModal = (
   );
   return { present, dismiss: dismissAll, bottomSheet };
 };
+
+/**
+ * Hook to generate a bottomSheet with a title, snapPoint and a component, that autosizes to the height of its content
+ * @param bottomSheetOptions
+ * @see {BottomSheetOptions}
+ * @param bottomPadding the bottom padding of the bottom sheet, defaul is 0
+ */
+export const useIOBottomSheetAutoresizableModal = (
+  {
+    component,
+    title,
+    snapPoint: originSnap,
+    footer,
+    onDismiss
+  }: BottomSheetOptions,
+  bottomPadding: number = 0
```

Comment:
Since the `snapPoint` using the `useIOBottomSheetAutoresizableModal` hook is auto-calculated by its content so if you pass any kind of value here it has no effect on the final result, what do you think if this parameter is omitted?

I suggest to create a new type which extends basically the BottomSheetOptions but omits the `snapPoint` props;
```typescript
type BottomSheetAutoresizableOptions = Omit<BottomSheetOptions, "snapPoint">;

```

Then it becomes as follow:

```suggestion
export const useIOBottomSheetAutoresizableModal = (
  {
    component,
    title,
    footer,
    onDismiss
  }: BottomSheetAutoresizableOptions,
  bottomPadding: number = 0
) => {

```